README.md

```diff
@@ -32,6 +33,13 @@ Custom recipes are Python code snippets that can be uploaded into Driverless AI
   * To disable all recipes: Set `enable_custom_recipes=false` in the config.toml, or add the environment variable `DRIVERLESS_AI_ENABLE_CUSTOM_RECIPES=0` at startup of Driverless AI. This will disable all custom transformers, models and scorers.
   * To disable new recipes: To keep all previously uploaded recipes enabled and disable the upload of any new recipes, set `enable_custom_recipes_upload=false` or `DRIVERLESS_AI_ENABLE_CUSTOM_RECIPES_UPLOAD=0` at startup of Driverless AI.
 
+* Enable automatic detection of forbidden or dangerous code constructs in a custom recipe with `custom_recipe_security_analysis_enabled = true`. Note the following:
+  * When `custom_recipe_security_analysis_enabled` is enabled, do not use modules specified in the banlist. Specify the banlist with the `custom_recipe_import_banlist` config option.
+    * For example: `custom_recipe_import_banlist = ["shlex", "plumbum", "pexpect", "envoy", "commands", "fabric", "subprocess", "os.system", "system"]` (default)
+  * Security analysis is only performed on recipes that are uploaded after the `custom_recipe_security_analysis_enabled` config option is enabled.
+  * To specify a list of modules that can be imported in custom recipes, use the `custom_recipe_import_allowlist` config option.
+  * The `custom_recipe_security_analysis_enabled` config option is disabled by default.
```

Comment:
I'd maybe extend and say that when `custom_recipe_security_analysis_enabled` is enabled, code is also checked for dangerous call like `eval(), exec()` or other insecure calls (regex patterns) defined in `custom_recipe_method_call_banlist` or other dangerous constructs defined as regex patterns in `custom_recipe_dangerous_patterns`